README.md

```diff
@@ -242,10 +242,12 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 * [simpletable](https://github.com/alexeyco/simpletable) - Simple tables in terminal with Go.
 * [tabby](https://github.com/cheynewallace/tabby) - A tiny library for super simple Golang tables.
 * [tabular](https://github.com/InVisionApp/tabular) - Print ASCII tables from command line utilities without the need to pass large sets of data to the API.
+* [tcell](https://github.com/gdamore/tcell) - Provides a cell based view for text terminals, like xterm. 
 * [termbox-go](https://github.com/nsf/termbox-go) - Termbox is a library for creating cross-platform text-based interfaces.
 * [termdash](https://github.com/mum4k/termdash) - Go terminal dashboard based on **termbox-go** and inspired by [termui](https://github.com/gizak/termui).
 * [termtables](https://github.com/apcera/termtables) - Go port of the Ruby library [terminal-tables](https://github.com/tj/terminal-table) for simple ASCII table generation as well as providing markdown and HTML output.
 * [termui](https://github.com/gizak/termui) - Go terminal dashboard based on **termbox-go** and inspired by [blessed-contrib](https://github.com/yaronn/blessed-contrib).
+* [tview](https://github.com/rivo/tview) - Provides commonly needed components for terminal based user interfaces.
```

Comment:
add more packages on this PR? update all links pls

Reply:
Well those are not my packages but happy to add the links for them. I did notice these were recommended by another (who was closed with a "send PR request please") so I thought I would add them. Both of these have the links on their README.md files so just a matter of copying them over.

## Tview Links

* github.com repo: https://github.com/rivo/tview
* godoc.org: https://godoc.org/github.com/rivo/tview
* goreportcard.com: https://goreportcard.com/report/github.com/rivo/tview
* coverage service link: https://gocover.io/github.com/rivo/tview

## Tcell Links

* github.com repo: https://github.com/gdamore/tcell
* godoc.org: https://godoc.org/github.com/gdamore/tcell
* goreportcard.com: https://goreportcard.com/report/github.com/gdamore/tcell
* coverage service link:  https://gocover.io/github.com/gdamore/tcell

Looks like both of them have some work to do to get up to par. I suppose we can leave this pull request open until they score better so people can know they need some help over there to polish them up. I'll try to get to it, but if not anyone can jump in and help them so they qualify to get posted.